orchestra/models/communication/models.py

```diff
@@ -102,7 +102,8 @@ class RequestCause(ChoicesEnum):
         RESTAFF = 'restaff'
 
     class Status(ChoicesEnum):
-        PROCESSING = 'processing'
+        SENDING_INQUIRIES = 'sending inquiries'
```

Comment:
enums are append-only. you can't insert values before others -> their IDs will get messed up

Reply:
depending on the comment below, you might be fine, as id = 1 (used to be COMPLETE) will now be WAITING_FOR_RESPONSES. we can discuss after you think both comments through.

Reply:
The previous COMPLETE status means that all the requests are sent out. In this case, WAITING_TO_RESPONSES means the same thing just more specific. The new enum COMPLETE is used to mark when there is the request is considered "closed" which is when 1) there is a winner for the task, or 2) all qualified workers declined the tasks.

Reply:
> enums are append-only, so it's dangerous to insert values before others -> their IDs will get messed up

This should be fine since we are changing just the labels and not the semantic meaning of the enums.